LanguageSpecification.md

```diff
@@ -951,6 +999,40 @@ end patch
 
 Assertions will generate warnings in default execution of the engine but `--ignore-assertions` can be executed in which case they will generate warnings on standard out instead.
 
+## Exports
+
+Josh supports exporting simulation results for analysis and visualization. Export attributes can be defined on patches and simulations to capture computed values at each timestep.
+
+### Export Attributes
+
+Export attributes are defined using the `export.` prefix and are evaluated at each timestep:
+
+```
+start patch Default
+
+  JoshuaTrees.init = create 10 count of JoshuaTree
+  
+  export.treeCount.step = count(current.JoshuaTrees)
+  export.averageAge.step = mean(current.JoshuaTrees.age)
+
+end patch
+```
+
+### Export Files
+
+Export destinations can be configured in simulations using `exportFiles` attributes:
```

Comment:
Please indicate that export values can also be provided to memory to be shown in the IDE as indicated in `/guides/hello.html`